lib_nbgl/include/nbgl_obj.h

```diff
@@ -234,6 +234,19 @@ typedef enum {
     PRESSED,   ///< the finger is currently pressing the screen
 } nbgl_touchState_t;
 
+#ifdef HAVE_HARDWARE_TOUCH_SWIPE
+/**
+ * @brief Hardware powered detected swipe states
+ */
+typedef enum {
+    HARDWARE_SWIPE_UP,
```

Comment:
To be consistant with touch state, this enum should be defined in seproxyhal_protocol.h (as SEPROXYHAL_TAG_FINGER_EVENT_TOUCH/SEPROXYHAL_TAG_FINGER_EVENT_RELEASE)